PAC/common/cip_tech_def.cpp

```diff
@@ -6404,14 +6404,13 @@ int cipline_tech_object::SCInitPumping( int what, int from, int where, int whatd
     rt_par_float[P_ZAD_FLOW] = operFlow;
     rt_par_float[P_VRAB] = operV;
     rt_par_float[P_OP_TIME_LEFT] = 0;
-    if (0 == operT)
+    if ( 0 == operT )
         {
-        float divider = rt_par_float[P_ZAD_FLOW];
-        if (0 == divider)
+        if ( operFlow == 0 )

```

Comment:
Или, @rartem, здесь лучше так сделать:
```suggestion
        if ( operFlow <= 0 )
```